GiraffeApp/Program.fs

```diff
@@ -35,27 +36,33 @@ let pageHandler (title: string)(view: XmlNode list)(next: HttpFunc)(ctx: HttpCon
     htmlNodes nodes next ctx
     
 [<CLIMutable>]
-type CountPayload = { Count: int }
+type IncrementForm = { Count: int }
 

```

Comment:
// I think this handler can be used to validate all form related handler
// I copied this one from https://github.dev/pimbrouwers/Giraffe.Antiforgery
// Checks the presence and validity of CSRF token and calls invalidTokenHandler on failure
// GET, HEAD, OPTIONS & TRACE always validate as true
let requiresCsrfToken (invalidTokenHandler : HttpHandler) : HttpHandler =
    fun (next: HttpFunc) (ctx : HttpContext) ->                                
        let antiFrg = ctx.GetService<IAntiforgery>()        
        (match antiFrg.IsRequestValidAsync(ctx) |> Async.AwaitTask |> Async.RunSynchronously with
        | true  -> next        
        | false -> invalidTokenHandler earlyReturn) ctx